src/sentry/api/base.py

```diff
@@ -176,9 +176,9 @@ def _create_api_access_log(self):
                 user=str(self.request.user),
                 token_type=token_name,
             )
-            api_access_logger.info(log_metrics)
+            api_access_logger.info("api.access", extra=log_metrics)
         except Exception:
```

Comment:
where in the above do we expect it to error out? could we catch more specific exceptions?

Reply:
I'm catching a generic exception because I don't want API requests to fail because of any logging error. As for where it could error out, most likely trying to access data members that don't exist, or key errors.